### DISCUSSION TITLE
WAG Contest Logging

### DISCUSSION THREAD

**Discussion by winnieXY** [wavelog/wavelog#1092] [General]
Hey,

I'm thinking about trying live contest logging for the upcoming WAG using wavelog. 

Is it somehow possible to have a number for the remote station and a exchange field (my DOK) for myself? (or vice versa for non-german operators)?

Or would it be the easiest solution to use the alphanumeric exchange field for both and just enter numbers here? this will work for me (as german operator) quite well i think, however non-german operators have to increase the number manually than (which could lead to errors). 

![grafik](https://github.com/user-attachments/assets/6e89d26f-4270-49cb-a10c-2e146ad9b202)
A option would be maybe to extend that menu by adding ([Send Exch, Recv Num],[Send Num, Recv Exch]).

What do you think?

**Comment by HB9HIL** [comment]
Use Serial Number and Exchange and just leave the fields empty you don't need :)
